checks/network.py

```diff
@@ -57,3 +57,19 @@ def checkNICSpeed(lines):
                     if 'GbE' in nic or 'Gigabit' in nic:
                         return [LEVEL_WARNING, "Slow Network Connection", "Your gigabit-capable network card is only connecting at 100mbps. This may indicate a bad network cable or outdated router / switch which could be impacting network performance."]
     return None
+
+
+x264stream_re = re.compile(r"stream")
+
+
+def checkDynamicBr(lines):
+    dynBrLines = search('Dynamic bitrate enabled', lines)
+    if (len(dynBrLines) > 0):
+        x264Lines = search('x264 encoder: ', lines)
+        for i in x264Lines:
+            if x264stream_re.search(i):
+                return [LEVEL_INFO, "Dynamic Bitrate", "Dynamic Bitrate is enabled. This does not resolve network issues, and will reduce bitrate (reduce quality) instead of dropping frames."]
```

Comment:
I feel this is worded a bit strangely. Something tagged as "Info" shouldn't prefix what it does do with what it doesn't. I'd be happy with just reusing the bot response here.

> Dynamic Bitrate is enabled. Instead of dropping frames when network issues are detected, OBS will automatically reduce the stream quality to compensate. The bitrate will adjust back to normal once the connection becomes stable.
> 
> In some (very specific) situations, Dynamic Bitrate can get stuck at a low bitrate. If this happens frequently, it is recommended to turn off Dynamic Bitrate in Settings -> Advanced -> Network.

Reply:
Understood. I feel the bot response is too long winded, and tried to keep it more precise, and at the same time trying to dispel a common misconception about how it works. How about this:

"Dynamic bitrate is enabled.  Instead of dropping frames when network issues are detected, OBS will automatically reduce the stream quality to compensate. You can disable it in Settings -> Advanced -> Network."

If not, we can do the whole thing, thats fine too :)

Reply:
I think it's fine to use longer messages in the log analyzer as this is one of the few places where errors are properly explained. In this particular case, i don't mind either way.